src/data/nav-items.yaml

```diff
@@ -205,15 +205,13 @@
       path: /how-to-contribute/add-ons
 - title: Updates
   pages:
-    - title: What’s new
-      path: /updates/whats-new
-    - title: Roadmap
-      path: /updates/roadmap
-    - title: Migrate to v10
-      path: /updates/v10-migration/overview
+    - title: Changelog and roadmap
+      path: /updates/changelog-and-roadmap
+    - title: Migration guide
+      path: /updates/migrate-to-v10/overview
```

Comment:
Did you intentionally change the path here? Path changes break any links in the wild, so shouldn't be taken lightly. We want to limit the rewrite rules as a last resort so they don't become unmanageable.

Considering most folks don't read the url, cosmetic changes or changes without a very strong motivation should be avoided all together.